src/Service/OpenAISpeechToTextService.php

```
@@ -0,0 +1,61 @@
+                'model' => $this->model,
+                'path' => $audioPath,
+                'size' => filesize($audioPath),
+            ]);
+
+            $response = $this->client->audio()->transcribe([
+                'model' => $this->model,
+                'file' => fopen($audioPath, 'r'),
+                'response_format' => 'json',
+            ]);
+
+            if (is_object($response) && isset($response->text)) {
```

PHPStan (2.1.15)

Call to function is_object() with OpenAI\Responses\Audio\TranscriptionResponse will always evaluate to true.


(function.alreadyNarrowedType)